Tests/Parser/fixtures/type-methods.yml

```
@@ -0,0 +1,46 @@
+### check format:
+### [isBot(), isMobile(), isDesktop(), isTablet(), isTV(), isWearable()]
+---
+-
+  user_agent: Mozilla/5.0 (Linux; Android 9; TEST-XXXX Build/PPR2.180905.006.A1; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/83.0.4103.120 YaBrowser/22.8.0.12 (lite) TV Safari/537.36
+  check: [false, false, false, false, true, false]
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 5.1.1; zh-CN; TEST-XXXXX Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/78.0.3904.108 Quark/5.3.3.191 Mobile Safari/537.36
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; HarmonyOS; TEST-XXXXX ; HMSCore 6.1.0.314) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.93 HuaweiBrowser/11.1.5.310 Mobile Safari/537.36
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Googlebot/2.1 (http://www.googlebot.com/bot.html)
+  check: [true, false, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (Linux; Android 4.4.2; Nexus 4 Build/KOT49H) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/33.0.1750.136 Mobile Safari/537.36
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (Linux; Android 4.4.3; Build/KTU84L) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/37.0.2062.117 Mobile Safari/537.36
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; WOW64; Trident/5.0)
+  check: [false, false, true, false, false, false]
+-
+  user_agent: Mozilla/3.01 (compatible;)
+  check: [false, false, false, false, false, false]
+-
+  user_agent: 'Mozilla/5.0 (Linux; U; Android 10; zh-CN; TEST-XXXX Build/HUAWEI) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/69.0.3497.100 UWS/3.22.1.228 Mobile Safari/537.36 AliApp(DingTalk/6.5.30) com.alibaba.android.rimet/25293052 Channel/227200 language/zh-CN abi/64 Pad/APad Hmos/1 UT4Aplus/0.2.25 colorScheme/light'
+  check: [false, true, false, true, false, false]
+### Mobile only browsers
+-
+  user_agent: Opera/9.80 (J2ME/MIDP; Opera Mini/9.5/37.8069; U; en) Presto/2.12.423 Version/12.16
+  check: [false, true, false, false, false, false]
+-
+  user_agent: 'Mozilla/5.0 (X11; U; Linux i686; th-TH@calendar=gregorian) AppleWebKit/534.12 (KHTML, like Gecko) Puffin/1.3.2665MS Safari/534.12'
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (Linux; Android 4.4.4; MX4 Pro Build/KTU84P) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/33.0.0.0 Mobile Safari/537.36; 360 Aphone Browser (6.9.7)
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (Macintosh; U; Intel Mac OS X 10_5_7; xx) AppleWebKit/530.17 (KHTML, like Gecko) Version/4.0 Safari/530.17 Skyfire/6DE
+  check: [false, true, false, false, false, false]
+-
+  user_agent: Mozilla/5.0 (Linux; U; Android 4.1.2; ru-ru; PMP7380D3G Build/JZO54K) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Safari/534.30
```

Guess that one contained some non unicode chars on purpose, so we can ensure they don't break anything.
```suggestion
  user_agent: Mozilla/5.0 (Linux; U; Android 4.1.2; ru-ru; PMP7380D3G Build/JZO54K) AppleWebKit/534.30 (KHTML, ÃÂºÃÂ°ÃÂº Gecko) Version/4.0 Safari/534.30
```

---

the fact is that linter swears at such symbols and cannot confirm validated